File: php/order/deleteOrder.php
<?php

/*
 * OrderApi.deleteOrder() will do just that.  It will delete an order.
 * You might find it more useful to reject an order rather than delete it in order to leave an audit trail.
 * However, deleting test orders will be useful to keep your order history tidy.  Still, any order
 * may be deleted.
 */

require_once '../vendor/autoload.php';
require_once '../samples.php';

$order_api = Samples::getOrderApi();

$order_id = 'DEMO-0008104390';
$order_api->deleteOrder($order_id);
echo 'Order was deleted successfully.';

File: python/order/delete_order.py
"""
OrderApi.deleteOrder() deletes an order. While rejecting orders is often preferred for audit trails,
deleting test orders can help maintain a tidy order history.
"""

from samples import api_client
from ultracart.apis import OrderApi
from ultracart import ApiException
import logging

# Configure logging
logging.basicConfig(level=logging.ERROR)
logger = logging.getLogger(__name__)

try:
    # Initialize Order API
    order_api = OrderApi(api_client())

    # Set order to delete
    order_id = 'DEMO-0009105482'

    # Delete order
    order_api.delete_order(order_id)
    print('Order was deleted successfully.')

except ApiException as e:
    logger.error(f"API Exception: {e}")
    print('Order could not be deleted.')